server/jetstream.go

```diff
@@ -260,7 +260,10 @@ func (s *Server) handleOutOfSpace() {
 				ID:   nuid.Next(),
 				Time: time.Now().UTC(),
 			},
-			Server: s.Name(),
+			Server:   s.Name(),
+			ServerID: s.ID(),
+			Stream:   stream,
```

Comment:
Not sure this matters, just chance TBH. The whole server is out of space.

Reply:
Stream? It's there if someone mounts a different volume onto the stream specific storage dir, so not the whole server always.

Reply:
This advisory signals the whole server shutdown jetstream. We could have the stream name who did it I guess..